README.md

```diff
@@ -47,14 +47,8 @@ rails server
 
 **Run tests**
 
-1. To run all tests in the project
-```ruby
-rspec
-``` 
-If you wanna run a certain test file
-```ruby
-rspec spec/test_file_path
-```
+- Under construction
+
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.